t/run/runenv_hashseed.t

```diff
@@ -76,9 +101,8 @@ for my $mode (@modes) {
             = my_runperl_and_capture( $run_opts, $print_keys );
 
         if ( $mode eq 'RANDOM' ) {
-            isnt( $out1, $out2,
-                "$descr results in different key order with the same keys"
-            );
+            # there is a 1/131072 chance that we got the same result.
+            $random_same++ if $out1 eq $out2;
         }
         else {
             is( $out1, $out2,
```

Comment:
Regarding the skip. I had the same thought but wanted to keep the patch minimally changed from @atoomic's original. But now that you suggest ill make the change. 

Regarding the running under DEBUGGING part, do you know a reliable way to tell if we are running under DEBUGGING in a test? If you do I am all for it, otherwise I'd rather leave it. I am not aware of a proper %Config key for that. (Its on my todo list to address actually, along with some other defines, but I am waiting on @neilb 's patch to add the taint stuff so I can crib from what he did.)

Reply:
> Regarding the running under DEBUGGING part, do you know a reliable way to tell if we are running under DEBUGGING in a test? If you do I am all for it, otherwise I'd rather leave it. I am not aware of a proper %Config key for that. (Its on my todo list to address actually, along with some other defines, but I am waiting on @neilb 's patch to add the taint stuff so I can crib from what he did.)

Oh, I assumed there would be an obvious way to test for DEBUGGING (short of trying to run `perl -DU - e1` and checking stderr), but if there isn't one right now I'm fine with deferring that until there is.

But I think it would already make sense to have it skip rather than pass if there's no data.